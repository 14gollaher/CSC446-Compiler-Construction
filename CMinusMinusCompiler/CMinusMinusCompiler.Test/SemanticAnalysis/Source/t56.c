int main()
{
  int x,y,z,x;
  return -1;
}
