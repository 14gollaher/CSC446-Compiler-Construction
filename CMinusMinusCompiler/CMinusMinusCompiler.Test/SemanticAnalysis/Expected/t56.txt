ERROR: Line 3 Duplicate lexeme "x" with depth "2" exists
Completed processing t56.c
